packages/vercel-edge/src/sdk.ts

```diff
@@ -59,6 +59,7 @@ export function getDefaultIntegrations(options: Options): Integration[] {
     linkedErrorsIntegration(),
     winterCGFetchIntegration(),
     consoleIntegration(),
+    // fixme: integration can be included - but integration should not add IP address etc
     ...(options.sendDefaultPii ? [requestDataIntegration()] : []),
```

Comment:
yeah I agree that we should probably handle this within the integration instead of the top-level decision here.